app/views/volunteers/show.html.slim

```diff
@@ -124,6 +124,12 @@ h3 Persönlicher Hintergrund
       tr
         td #{t_attr(:external)} / #{t_attr(:internal)}
         td= @volunteer.external? ? t_attr(:external) : t_attr(:internal)
+        tr
+          td= t('registered_by')
+          - if @volunteer.self_applicant?
```

Comment:
nest the conditional below the td element:

```slim
td
  - if xy
    = some content
  - else
    = other content
```

Reason: You can see that on the indent error. If you have the slim html element not nested inside the conditional, you dont get that much indent confusion...